vector.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: vector
+  version: "0.34.0"
+  epoch: 0
+  description: End-to-end observability data pipeleine
+  copyright:
+    - license: Mozilla Public License 2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - cyrus-sasl-dev
+      - librdkafka
+      - librdkafka-dev
+      - openssl
+      - openssl-dev
+      - perl
+      - pkgconf-dev
+      - protobuf-dev
+      - rust
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/vectordotdev/vector
+      tag: v${{package.version}}
+      expected-commit: c909b660ca3cf3cc1c6d9cd7880cef8b61b2b426
+
+  - runs: |
+      sed -i '7s/deny/allow/' src/lib.rs
+      cargo build --release --no-default-features --features default
```

Comment:
I don't know a lot about `cargo`, but the flags `--no-default-features` and `--features default` seem conflicting. Is there a reason for that?